### ISSUE TITLE
Random crashes 

### ISSUE THREAD

**Issue by johng** [omnilaboratory/obd#14]
```
2020/07/16 21:57:16 schedule.go:31: timer 10m 2020-07-16 21:57:16.001173196 +0100 BST m=+601.028695263
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x8c5592]

goroutine 54 [running]:
github.com/omnilaboratory/obd/service.sendRdTx()
        /home/johng/repos/obd/service/schedule.go:99 +0x152
github.com/omnilaboratory/obd/service.(*scheduleManager).StartSchedule.func1()
        /home/johng/repos/obd/service/schedule.go:32 +0x126
created by github.com/omnilaboratory/obd/service.(*scheduleManager).StartSchedule
        /home/johng/repos/obd/service/schedule.go:21 +0x35
```

There doesn't seem to be consistent trigger.

**Comment by iamgamelover** [comment]
Is this problem repeated every time? Could you describe the operation process?

**Comment by iamgamelover** [comment]
Fixed at the commit: a1f99d2225ce9701fb083a0e519d43519a594e72

**Comment by johng** [comment]
I couldn't say what triggered it
